src/components/LoginForm..jsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react';
+import {Tab, Tabs, TextField, Stack, Button, Container } from "@mui/material";
+import { Box } from '@mui/system';
+
+const passwordMatchingText = {
+    error: "Passwords do not match",
+    valid: ""
+}
+function LoginForm() {
+    const [currentTabIndex, setCurrentTabIndex] = useState(0);
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+    const [passwordVerification, setPasswordVerification] = useState('');
+    const [passwordMatchingMsg, setPasswordMatchingMsg] = useState(passwordMatchingText.valid)
+    
+    const handleTabChange = (event, tabIndex) => {
+        setCurrentTabIndex(tabIndex);
+        setEmail('')
+        setPassword('')
+        setPasswordVerification('')
+        setPasswordMatchingMsg(passwordMatchingText.valid)
+    };
+
+    const handleEmailChange = (event) => {
+        setEmail(event.target.value)
+    }
+
+    const handlePasswordChange = (event) => {
+        setPassword(event.target.value)
+        setPasswordMatchingMsg(passwordMatchingText.valid)
+    }
+
+    const handlePasswordVerificationChange = (event) => {
+        setPasswordVerification(event.target.value)
+        setPasswordMatchingMsg(passwordMatchingText.valid)
+    }
+
+    /*
+    TODO:
+    - Handle api status code (200 vs != 200)
+    */
+    const onLogin = (event) => {
+        event.preventDefault()
+        alert("Welcome " + email)
+
+    }
+
+    /*
+    TODO:
+    - Handle api status code (200 vs != 200)
+    */
+    const onCreateAccount = (event) => {
+        event.preventDefault()
+        if (password !== passwordVerification) {
+            setPasswordMatchingMsg(passwordMatchingText.error)
+        }
+    }
+
+    return (
+        <Container sx={{width: "25%",border: "1px solid gray" }}>
+            <Tabs value={currentTabIndex} sx={{ borderBottom: 1}} onChange={handleTabChange}>
+                <Tab label="Login" />
+                <Tab label="Sign Up" />
+            </Tabs>
+            {  
+                currentTabIndex === 0 && (
+                    <form onSubmit={onLogin}>
+                    <Box sx={{ p: 3}} >
```

Comment:
What is the equivalent of p: 3 to CSS?

Reply:
Padding! 

P3

![image](https://github.com/secure-password-manager/PasswordManagerWeb/assets/76857371/c1d41375-cb09-47c5-8eb3-d464148b130d)

P10

![image](https://github.com/secure-password-manager/PasswordManagerWeb/assets/76857371/ad75f6be-df56-433c-b571-aa0f961518ae)

No padding

![image](https://github.com/secure-password-manager/PasswordManagerWeb/assets/76857371/13472b2f-4213-4c36-9f42-b258bf4ffdad)